config/locales/client.en.yml

```
@@ -35,8 +31,34 @@ en:
         label:
           active: Active
           not_active: Not Active
-      discovery:
-        description: Follow %{category_name} on services that support ActivityPub using
+      category_nav:
+        label: Federation
+        description: Federation for %{category_name}
+      banner:
+        text: Follow %{category_name} on any service that supports federation
+        mobile_text: Federated
+        public_full_topic: Each post of each topic will be published publicly via ActivityPub.
+        public_first_post: The first post of each topic will be published publicly via ActivityPub.
+        followers_only_full_topic: Each post of each topic is published to ActivityPub followers only.
+        followers_only_first_post: The first post of each topic is published to ActivityPub followers only.
+      handle:
+        label: Other
+        description: find on any service that supports ActivityPub
+      follow:
+        label: Follow
+        title: Follow %{name}
```

```suggestion
        title: Follow %{name} via ActivityPub
```

Makes for a slightly larger modal (more breathing room) but also nice to use the technical term (ActivityPub) here since we're not using it in the "Federation" screen.